Cargo.toml

```diff
@@ -11,10 +11,9 @@ license-file = "LICENSE"
 [dependencies]
 clap = "3.0.0-beta.2"
 serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
 serde_yaml = "0.9"
-serde_with = {version = "2.0.0", features = ["json"]}
+serde_with = { version = "2.0.0", features = ["json"] }
```

Comment:
would we still need json here?

Reply:
It seems to me like it was in fact _never_ needed 🤣